### PR TITLE
Shell improvements

### PR DESCRIPTION
**Pull request by erikarvstedt**
We should include this in the next release.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Updating
 ---
-In your deployment directory, enter the nix shell with `nix-shell` and run
+In your deployment directory, enter the nix shell with `nix-shell` and run the
+following to update `nix-bitcoin-release.nix`:
 
 ```
-fetch-release > nix-bitcoin-release.nix
+update-nix-bitcoin
 ```
```
Comment (nixbitcoin) [inline docs/usage.md:9]: Should we also put a note about deploying, or is that self-explanatory?

Comment (erikarvstedt) [reply docs/usage.md:9]: It think it's self-evident due to the shell output:
```
Fetched and verified release 0.0.50
Updated nix-bitcoin-release.nix
```

Comment (nixbitcoin) [reply docs/usage.md:9]: Ok

File: helper/fetch-release [hunk 2]
```diff
@@ -28,7 +28,7 @@ curl --silent -L -O $baseUrl/nar-hash.txt.asc
 
 # Verify signature for nar-hash
 gpg --homedir $GPG_HOME --verify nar-hash.txt.asc &> /dev/null || {
-    echo "Error: Signature verification failed. Please open an issue in the project repository."
+    >&2 echo "Error: Signature verification failed. Please open an issue in the project repository."
```
Comment (nixbitcoin) [inline helper/fetch-release:31]: Running `fetch-release > nix-bitcoin-release.nix` still wipes the `nix-bitcoin-release.nix`

Comment (erikarvstedt) [reply helper/fetch-release:31]: I've now completely removed `fetch-release > nix-bitcoin-release.nix` from the docs.

**Review by nixbitcoin — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

ACK f7848eb219209f5206e6ae336298e1d4a6bd5bdb
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEiDOlIkszv4ojhh+OtgROy6La5dAFAmE/UwoACgkQtgROy6La
5dDcww/9EvhlO85tla4ow2WQFEgfr+hXwbocxNSN1RruRO7OaF+j4FVjgXYrJYIJ
BwQvfwVCuU3A9rYqwHDv/CYzA5IslFklnG6EKcA46N917Rw9ySKJo+6BNaIgF4SC
B+aUwXcKRUwVJzpujKQDO1Xp1O72ATpD+1YiP0NCpZO1KF4QeT0DdnDNpkV4qpdO
pVWhtitbFZ8b80dfWc8/ujZuhSjVc5QHRdR+yTdl3ox+T1f5JUtT4syCj6CXsKYC
mpmYYny3Sb25juVZwkpD96gjfgGtO2tlMLeQSJ/QsoTDFQKR3y5whjuouQ0zN+Uw
ObkNbVLAUJgoakjAhqXC4DDcyRCg4zHnpbortez2/RIhlKIYkXoYLpNE05cpprPP
bHYvbrjpU0s4zsz2DGbLvV/UexbRR8UZ9shB0jEqC4ws7kwIjkS04NqM8YUqfYC6
qMQrJIIa49TLIFL4Bo0VdEUby6lrz2jn+2zGbpN/br6R5HubbqQWlrdNQQA+WGxp
MuvZ1sVTfHoDtl/JflGXnoQO5dFXDbc5uIAG87Be8OYFs0rDS1zJ7P2ImIRH2bK3
IbHDI9oedJeNMQmXhFuQIyHy3lHkLwviHk7Ei/a1zD2WaHSDDY3fe5J4/prT3u8T
svzbHR0Q0P9HkuXhJVjK3Nrv8i0SOWdqKsNjcc4LCefHmmH57TY=
=QO/R
-----END PGP SIGNATURE-----
```

**Review by nixbitcoin — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

ACK a04125c8926ba6f668f49c64743057ada28e757d
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEiDOlIkszv4ojhh+OtgROy6La5dAFAmE/U50ACgkQtgROy6La
5dBC3w//Ssr6MXWx2c12v+IucYMy8VGm3KgRSGhQo5NT0o6WqCHXAtaTemVNYPS/
HQvD/mM8sqeV6RoJawCvm4J2zOwrcD1hdRQWkPZogX1aPeaokVlSOy644+D676bb
0ICfpLyZaLWykl4CHFTVykG8YpmTiXIVM418AyEaht7m3KkNTVmKnp6ql+RNcEm4
Tn3GXgDWTL7jbJNyEYO+Hx77oYjORhYVn6Z/hyIQwVvpwGBcbgCGLrpmabZ28Pbr
eiGqt+i5Iy++ZgMuIAmPxGVJ9kBwXewIeg4WKU7VuAcUDd9lsbQEEnwm3/IdRtr2
3cEthkM5+3HaRJTCGr/6y0l8fkogGpRFxuAgEA0VGfnHjhEaKk6jsVTMUCNZXeqk
ehrMmUBDpJtsoOHrqPNqbXMfZG0f3d4HPJMsRGpCRcDo4vcutnHFy+fkqLtIZyzT
68QaiAEbTsosJwG8iaxaxjLYB9iud+zr9f2rJ/pDRba8nlGvoWhh4G0UJv4Dnxuz
ImrRVb1XeAq2y6bESj9gfUkS2acPjzwUGIVLT6GrecJKMGaKpq2MlTK2WZxH6Ifv
LedC/dyj5EqQoCRkIDI6wU+L5r99+PcOsv8fQjyUO/KgzXnJBoXfNBK7r5HCkNp2
raT7CVhLLNuzUQeKRU1QokBP1jqHQw3UPIvJ4SZep06ooKQ3fNA=
=EHYJ
-----END PGP SIGNATURE-----
```

**Comment by erikarvstedt** [comment]
I've amended `fetch-release: write error messages to stderr` to also show curl error messages.

**Review by jonasnick — APPROVED** [review]
ACK b49c74545fb57a429339dc448bb2c0f6a1cb3734

**Pull request merged**